packages/react-accordion/src/components/Accordion/useAccordion.ts

```diff
@@ -63,7 +68,7 @@ function initializeUncontrolledOpenItems({
     }
     return [defaultOpenItems];
   }
-  return collapsible ? [] : [0];
+  return collapsible ? [] : [];
```

Comment:
Yes, thx @varholak-peter ! This definitely needs a TODO comment. basically this behavior is not working properly since modifications due to performance issue.